test/auro-background.test.js

```diff
@@ -29,97 +29,97 @@ describe('auro-background', () => {
       <auro-background></auro-background>
     `);
 
-    await expect(el.shadowRoot.innerHTML).contains(`--inset:var(--auro-size-none)`);
-    await expect(el.shadowRoot.innerHTML).contains(`--insetSm:var(--inset)`);
-    await expect(el.shadowRoot.innerHTML).contains(`--insetMd:var(--insetSm)`);
-    await expect(el.shadowRoot.innerHTML).contains(`--insetLg:var(--insetMd)`);
+    await expect(el.shadowRoot.innerHTML).contains(`--inset: var(--auro-size-none)`);
```

Comment:
Shouldn't this have been updated to use a `--ds` value?

Reply:
I am not sure about this, it does refer to this render function though, should I make the changes on this also?

<img width="669" alt="image" src="https://github.com/AlaskaAirlines/auro-background/assets/124668226/980659f5-ffb3-4949-9570-9381a4f806bd">